Extensions/Taxonomies/ITaxonomySetupService.cs

```diff
@@ -20,5 +20,13 @@ public interface ITaxonomySetupService : IDependency
         /// <param name="taxonomyName">Names of the Taxonomy to create.</param>

```

Comment:
Úgy látom az előbbi plural vs. singular hiba innen jön, akkor ezt is javítsd légyszi.